packages/main/src/TableGrowing.hbs

```diff
@@ -1,19 +1,17 @@
-{{#if _hasGrowingButton}}
```

Comment:
Hi Nayden, just to learn, why this if check is removed? Do we always need to render something? Is it wrong conditionally not render something?

Reply:
There were cases where the shadow DOM existed, but nothing was rendered inside. This was incorrect because any component with a shadow DOM should have content inside it; otherwise, it should be marked as abstract. If it needs to be hidden from the DOM, that should be handled by the parent component (e.g., table).